pydocx/export/base.py

```diff
@@ -547,7 +573,5 @@ def export_textbox_content(self, textbox_content):
         return self.yield_nested(textbox_content.children, self.export_node)
 
     def export_markup_compatibility_alternate_content(self, alternate_content):
-        return self.yield_nested(alternate_content.children, self.export_node)
-
-    def export_markup_compatibility_fallback(self, fallback):
-        return self.yield_nested(fallback.children, self.export_node)
+        if self.first_pass:
+            self.alternate_contents.append(alternate_content)
```

Comment:
I'd rather not edit the structure of the XML while we are parsing it. I think it makes more sense to do the altering in between passes.